docs/install.md

```diff
@@ -0,0 +1,142 @@
+# Install Instructions
+
+Once the install steps are complete [MAVIS](http://mavis.bcgsc.ca) is ready to be run.
+See the MAVIS [tutorial](https://mavis.readthedocs.io/en/latest/tutorials/mini) to learn about running MAVIS.
+
+For either install option you will want to install the main Snakefile. It is best to use a tag to
+specify the version of interest but you can download the latest version from the master branch
+
+```bash
+wget https://raw.githubusercontent.com/bcgsc/mavis/master/Snakefile -O Snakefile
+```
+
+## Install for Docker/Singularity
+
+The simplest way to use MAVIS is via Singularity. The MAVIS docker container used
+by singularity will take care of installing the aligner as well.
+
+```bash
+pip install -U setuptools pip
+pip install mavis_config  # also installs snakemake
+```
+
+Now you will run mavis via Snakemake as follows
+
+```bash
+snakemake \
+    -j <MAX JOBS> \
+    --configfile <YOUR CONFIG> \
+    --use-singularity \
+    -s Snakefile
+```
+
+## Install (Python Only)
+
+MAVIS can also be run with just python. However you will need to install the aligner(s) required
+by MAVIS separately and ensure they are availble on the default PATH variable when MAVIS is run
+
+### 1. Install Aligner
+
+In addition to the python package dependencies, [MAVIS](http://mavis.bcgsc.ca) also requires an aligner to be installed.
+Currently the only aligners supported are [blat](https://mavis.readthedocs.io/en/latest/glossary/#blat) and [bwa mem](https://mavis.readthedocs.io/en/latest/glossary/#bwa).
+For MAVIS to run successfully the aligner must be installed and accessible on the path.
```

Comment:
Could these be packaged with the docker / singularity as well, eventuallly?

Reply:
They already are. This section is just in case we have users that don't have docker